docs/content/reference/cli/authelia-scripts/authelia-scripts_suites_external_list.md

```
@@ -0,0 +1,53 @@
+
+## authelia-scripts suites external list
+
+List available external suites
+
+### Synopsis
+
+List available external suites.
+
+External suites can be run with the authelia-scripts suites external test [suite] command.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)